src/scripts/main.js

```diff
@@ -1,3 +1,67 @@
 'use strict';
 
-// write code here
+const names = document.querySelector('thead > tr > th:nth-child(1)');
+const position = document.querySelector('thead > tr > th:nth-child(2)');
+const age = document.querySelector('thead > tr > th:nth-child(3)');
+const salary = document.querySelector('thead > tr > th:nth-child(4)');
+const tableBody = Array.from(document.querySelectorAll('tbody > tr'));
+const tbody = document.querySelector('tbody');
+
+names.addEventListener('click', (e) => {
+  e.stopPropagation();
+
+  tableBody.sort((a, b) => {
+    const textA = a.querySelector('td:nth-child(1)').textContent.trim();
+    const textB = b.querySelector('td:nth-child(1)').textContent.trim();
+
+    return textA.localeCompare(textB);
+  });
+
+  tbody.innerHTML = '';
+  tableBody.forEach((row) => tbody.appendChild(row));
+});
+
+position.addEventListener('click', (e) => {
+  e.stopPropagation();
+
+  tableBody.sort((a, b) => {
+    const textA = a.querySelector('td:nth-child(2)').textContent.trim();
+    const textB = b.querySelector('td:nth-child(2)').textContent.trim();
+
+    return textA.localeCompare(textB);
+  });
+
+  tbody.innerHTML = '';
+  tableBody.forEach((row) => tbody.appendChild(row));
+});
+
+age.addEventListener('click', (e) => {
+  // e.stopPropagation();
```

Comment:
The `stopPropagation` method is commented out for the age column. If this is intentional, ensure that no parent elements are affected by the click event. Otherwise, consider uncommenting it to maintain consistency with other event listeners.